examples/secured-service-with-jwt/secured_service_with_jwt.description

```diff
@@ -1,15 +1,15 @@
 // A service can be secured using JWT and optionally, by enforcing
 // authorization. The JWT auth provider verifies the signature of the JWT
 // token sent in the Authorization header. For this, Ballerina should trust the
-// JWT issuer (must have the certificate of the JWT issuer in the Ballerina
+// JWT issuer (the certificate of the JWT issuer must be present in the Ballerina
 // certificate truststore).
 // Ballerina uses the concept of scopes for authorization. A resource
-// declared in a Service can be bound to one/more scope(s). In the config
+// declared in a service can be bound to one/more scope(s). In the config
 // auth provider, scopes are mapped to users.
-// In the authorization phase, the scopes of the resource is compared against
-// the scopes mapped to the user for at least one match between the two sets.
-// Here, scope can be included in the JWT, via an attribute named 'scope',
-// or the scopes for a user can be set in a sample-users.toml file as below:
+// In the authorization phase, the scopes of the resource are compared against
+// the scopes mapped to the user for atleast one match between the two sets.
```

Comment:
Why change `at least` -> 'atleast`?

Reply:
You're right. "at least" is correct as well. Fixed with https://github.com/ballerina-platform/ballerina-examples/pull/762/commits/983e8979cb3a062076001ea9f5c0507a22cb1af9